src/openforms/forms/api/serializers/form_variable.py

```diff
@@ -118,6 +135,28 @@ class Meta:
         # applying # this validation on the whole collection.
         validators = []
 
+    def validate_service_fetch_configuration(self, value):
+        if value is None:
+            return value
+        config_instance = None
+        if config_id := value.get("id"):
+            try:
+                config_instance = ServiceFetchConfiguration.objects.get(id=config_id)
```

Comment:
`ServiceFetchConfiguration.objects.filter(id=config_id).exists()` is more idiomatic (it returns a bool and does a `SELECT 1 FROM` query

Reply:
But don't we need the instance to pass to the `ServiceFetchConfigurationSerializer` in line 155?

Reply:
yes, I'm an idiot :grimacing: